gramps/gui/views/treemodels/citationbasemodel.py

```diff
@@ -54,26 +56,26 @@
 # -------------------------------------------------------------------------
 # These are the column numbers in the serialize/unserialize interfaces in
 # the Citation object
-COLUMN_HANDLE = 0
-COLUMN_ID = 1
-COLUMN_DATE = 2
-COLUMN_PAGE = 3
-COLUMN_CONFIDENCE = 4
-COLUMN_SOURCE = 5
-COLUMN_CHANGE = 9
-COLUMN_TAGS = 10
-COLUMN_PRIV = 11
+COLUMN_HANDLE = "handle"
```

Comment:
Would the COLUMN_XXX be clearer if called KEY_XXX since they are now the key in to a Python dictionary?

Reply:
See note. But I think using the actual schema name is clearer.